src/main/java/org/maxgamer/quickshop/shop/ContainerShop.java

```diff
@@ -788,7 +865,6 @@ public void onLoad() {
                 }
             }
         }
-        this.checkDisplay();
```

Comment:
Why It's was being removed?

Reply:
Excellent catch! I have no recollection of removing that but it was a weird source of bugs where the items would not load in correctly and I never even though to look in the onLoad method, I literally had it folded in my IDE. It's been added back, to superb effect.